code/modules/cargo/exports/organs.dm

```diff
@@ -75,14 +78,14 @@
 	if(C.preserved)
 		return ..() * 2
 	return ..()
-	
+
 // Kitty Organs.
-	
+
 /datum/export/organ/cat/ears
```

Comment:
cat shit can be toys. sex toys, I guess.

Reply:
counts as organs iirc, been two months since i touched it